__tests__/components/CalendarButton.test.tsx

```diff
@@ -1,9 +1,9 @@
 import CalendarButton from '../../src/components/CalendarButton';
 import renderer from 'react-test-renderer';
 import React from 'react';
-import { APIEvent } from '@knighthacks/hackathon';
+import { APIEventData } from '@knighthacks/hackathon';
 
-const testJSON: APIEvent = {
+const testJSON: APIEventData = {
```

Comment:
so usually tests like this are data driven. That is, the test data like this event you made is exported to an external CSV. For now this is fine though.

Reply:
That would involve parsing the csv to make the event data object which isn't something we have time for. This will be bumped to the next team.